src/services/token.service.ts

```diff
@@ -183,4 +212,10 @@ export class TokenService implements TokenServiceAbstraction {
 
         return decoded.iss as string;
     }
+
+    private async skipTokenStorage(): Promise<boolean> {
+        const timeout = await this.storageService.get<number>(ConstantsService.vaultTimeoutKey);
+        const action = await this.storageService.get<string>(ConstantsService.vaultTimeoutActionKey);
+        return timeout != null && action === 'logOut';
```

Comment:
Is timeout = null the "Never" case?

Reply:
^ That is correct!

Reply:
Here is a breakdown of the values from each platform:

**Browser**
- `null` -> Never
- `0` -> Immediately

**Desktop**
- `null` -> Never

**Web Vault**
- `null` -> Never